### DISCUSSION TITLE
virt-manager failing on macOS M1 Monterey after most recent update

### DISCUSSION THREAD

**Discussion by jorgy80236** [Homebrew/discussions#3894] [Everyday usage] [answered]
A couple of days ago, I did a `brew upgrade` and now virt-manager is failing.  FIrst, the formalities:

```
$ brew doctor
Your system is ready to brew.
```

```
$ brew config
HOMEBREW_VERSION: 3.6.9
ORIGIN: https://github.com/Homebrew/brew
HEAD: 92e3ea0ecc219510d850c639c035bf0cdc6903f3
Last commit: 35 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 47c924656ffea421487417ecd363bd60a98900f8
Core tap last commit: 2 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.oPqqpxjRC1/org.xquartz:0
HOMEBREW_EDITOR: /usr/bin/vim
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.37.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.79.1 => /usr/bin/curl
macOS: 12.6.1-arm64
CLT: 14.1.0.0.1.1666437224
Xcode: 14.1
Rosetta 2: false
```

```
  Model Name:	MacBook Pro
  Model Identifier:	MacBookPro18,3
  Chip:	Apple M1 Pro
  Total Number of Cores:	10 (8 performance and 2 efficiency)
  Memory:	16 GB
  System Firmware Version:	8419.41.10
  OS Loader Version:	7459.141.1
```

This is a fully patched Monterey Apple Silicon box running an up-to-date brew.   When I start virt-manager, I get a new warning, but it seems to start okay.

```
$ virt-manager 
/opt/homebrew/Cellar/virt-manager/4.1.0_3/libexec/lib/python3.10/site-packages/requests/__init__.py:109: RequestsDependencyWarning: urllib3 (1.26.12) or chardet (None)/charset_normalizer (3.0.0) doesn't match a supported version!
  warnings.warn(
```

However, when I try to pull up a guest on a remote host, it crashes trying to access the console.  No error message on console is given.    Here is the output from the macOS crash dialog:

https://gist.github.com/jorgy80236/08c37d8b4d4e9aa910dc9cdc6c44d557

It doesn't happen on every host I've tested, but reliably crashes on the ones that don't work.

Fedora Core 34
libvirt-7.0.0-8.fc34.x86_64
Crash:   no

Rocky 8.6
libvirt-8.0.0-5.4.module+el8.6.0+1000+18e3b59f.x86_64
Crash:  yes - used to work

Rocky 8.6
libvirt-8.0.0-5.5.module+el8.6.0+1052+ff61d164.x86_64
Crash:  yes - used to work, and several servers on this rev all crash

It used to work everywhere, and it looks like there is something that has been introduced in the most recent virt-viewer brew update that doesn't like the version 8.0.0 of libvirt on the server.   I will be pleased to furnish any more information required.  Thanks,

jorgy

**Comment by SMillerDev** [comment]
Could you make an issue for this?

**Comment by jorgy80236** [comment]
Sure thing.

**Comment by jorgy80236** [comment, marked as answer]
Created issue #115413.  Thanks.
